tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -106,7 +106,7 @@
     @Path("/status")
     @Produces(MediaType.APPLICATION_JSON)
     @Status
-    @LRA(LRA.Type.NOT_SUPPORTED)
+    @LRA(value = LRA.Type.NOT_SUPPORTED, terminal = false)
```

Comment:
Does it matter what the value for `terminal` is when NOT_SUPPORTED?

Reply:
+1